packages/valory/skills/decision_maker_abci/behaviours/sampling.py

```diff
@@ -118,7 +155,6 @@ def async_act(self) -> Generator:
         """Do the action."""
         with self.context.benchmark_tool.measure(self.behaviour_id).local():
             idx = self._sample()
-            self.store_bets()
```

Comment:
Why removed?

Reply:
This got removed because if the agent crashes or is stopped in between sampling and the bet_placement/blacklisting the n_bets becomes inaccurate and you then face an issue in using bet.prediction_response.vote.